indico/legacy/webinterface/rh/base.py

```diff
@@ -551,13 +551,13 @@ def _do_process(self, profile):
         return profile_name, res
 
     def _process_success(self):
-        Logger.get('requestHandler').info('Request successful')
-        # request is succesfull, now, doing tasks that must be done only once
+        logger = Logger.get('requestHandler')
+        logger.info('Request successful')
+        # request is succesfull, let's get rid of temporary files
         try:
-            GenericMailer.flushQueue(True)  # send emails
             self._deleteTempFiles()
```

Comment:
This is actually obsolete and can go away - `self._tempFilesToDelete` is always an empty list in both RH and ServiceBase